controls/cis_rhel8.yml

```diff
@@ -2529,18 +2510,28 @@ controls:
       - file_owner_etc_passwd
       - file_permissions_etc_passwd
 
-  - id: 6.1.4
-    title: Ensure permissions on /etc/shadow are configured (Automated)
+  - id: 6.1.2
+    title: Ensure permissions on /etc/passwd- are configured (Automated)
     levels:
       - l1_server
       - l1_workstation
     status: automated
     rules:
-      - file_owner_etc_shadow
-      - file_groupowner_etc_shadow
-      - file_permissions_etc_shadow
+      - file_groupowner_backup_etc_passwd
+      - file_owner_backup_etc_passwd
+      - file_permissions_backup_etc_passwd
 
-  - id: 6.1.5
+  - id: 6.1.3
+    title: Ensure permissions on /etc/security/opasswd are configured (Automated)
```

Comment:
I will note the this is a deviation from the benchmark, but I think it makes sense.

Reply:
Yes, the description is ok but the title is not correct in the policy. I saw it was already updated for a future release: https://workbench.cisecurity.org/sections/2299710/recommendations/3684693